Updated README (#92)
nextcloud/notes-ios: README.md

<div align="center">
    <img src="Brand/Custom.xcassets/AppIcon.appiconset/1024.png" alt="App Icon" width="256" height="256" />
    <h1>Nextcloud Notes for iOS and iPad OS</h1>
</div>

Nextcloud Notes is a native client for the [Nextcloud](https://nextcloud.com "Nextcloud") [Notes](https://apps.nextcloud.com/apps/notes "Notes") app.
It allows you to add, edit, and delete your notes on iPhone and iPad.
Plain Markdown editing and rich text editing is supported.
Notes can be exported to several formats.

> With Nextcloud Notes the notes can also be organized in categories.

Reviewer: ```suggestion
Notes can be organized in categories.
```

## Download on the App Store

[![Download on the App Store](./README/download-on-the-app-store.svg)](https://itunes.apple.com/app/id813973264)

You can download Nextcloud Notes directly from the [App Store](https://apps.apple.com/app/nextcloud-notes/id813973264).

> If you are viewing this on a different device, use the camera in your iPhone or iPad to scan the QR code below to open it in the App Store.

Reviewer: ```suggestion
Scan the QR code below to download the app from the App Store.
```
